pkg/workloads/cortex/serve/serve.py

```diff
@@ -153,12 +153,10 @@ def predict(request: Any = Body(..., media_type="application/json"), debug=False
 
     debug_obj("payload", request, debug)
     prediction = predictor_impl.predict(request)
-    debug_obj("prediction", prediction, debug)
 
-    try:
-        json_string = json.dumps(prediction)
-    except:
-        json_string = util.json_tricks_encoder().encode(prediction)
+    json_string = json.dumps(prediction)
```

Comment:
Do you think it's worth adding our own error message if this throws an exception or is the message that's shown clear enough for the user?